### PR TITLE
chore: bump wasmvm

### PR DESCRIPTION
**Pull request by the-frey**
Updating wasmvm to the latest version. Child's stuff.

### CODE REVIEW

**Review by Reecepbcups — CHANGES_REQUESTED** [review]
Actually we do not need to do a replacement since we already use the wasmvm

**Review by Reecepbcups — APPROVED** [review]

**Comment by the-frey** [comment]
Good spot.

**Review by dimiandre — APPROVED** [review]
lgtm

**Review by JakeHartnell — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -34,7 +34,7 @@ require (
 	github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 // indirect
 	github.com/99designs/keyring v1.2.1 // indirect
 	github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d // indirect
-	github.com/CosmWasm/wasmvm v1.2.1 // indirect
+	github.com/CosmWasm/wasmvm v1.2.3 // indirect
```
Comment (webmaster128) [inline go.mod:37]: Is this safe to never revert back? I mean all your direct dependencies only ask for 1.2.1 (`github.com/CosmWasm/wasmd v0.31.0`). But I'm not an expert wrt. Go build systems and `indirect` is a mystery to me.

**Review by webmaster128 — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (robert-zaremba) [reply go.mod:37]: In case of doubt, always check `go.sum` - this is what is being used.

**Pull request merged**